src/main/java/io/vertx/lang/groovy/ConversionHelper.java

```diff
@@ -58,9 +59,13 @@ private static Object toJsonElement(Object obj) {
   }
 
   public static JsonObject toJsonObject(Map<String, Object> obj) {
-    return obj == null ? null : new JsonObject(obj.entrySet().stream().collect(Collectors.toMap(
-      Map.Entry::getKey, entry -> toJsonElement(entry.getValue())
```

Comment:
This is point in the code where the NPE is being thrown, and so far I have been unable to determine why.